src/core/cita_bft.rs

```diff
@@ -1401,6 +1407,8 @@ impl Bft {
                 });
             }
         } else if tminfo.step == Step::PrecommitAuth {
+            // If consensus doesn't receive the result of block verification in a specific
+            // time-frame，use the original message to construct a request, then resend it to auth.
```

Comment:
Chinese comma

Reply:
I don't think so.

Reply:
Oh, I see it, I will update this PR after all discussions are finished.